parser.go

```diff
@@ -285,33 +299,60 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 	parserBufferPeekResult, _ := p.buf.Peek(bufferSize)
 	peekBuffer := bytes.NewBuffer(parserBufferPeekResult)
 
+	indentSize := 0
 	for {
 		peekData, peekErr := peekBuffer.ReadBytes('\n')
 		if peekErr != nil {
 			if peekErr == io.EOF {
+				p.debug("readPythonMultilines: failed to peek")
+				p.debug("readPythonMultilines: peekData is: '"+string(peekData)+"'")
+				p.debug("readPythonMultilines: value is: '"+line+"'")
 				return line, nil
 			}
+			p.debug("readPythonMultilines: failed to peek, returning error")
 			return "", peekErr
 		}
 
+		p.debug("readPythonMultilines: parsing '"+string(peekData)+"'")
+
 		peekMatches := pythonMultiline.FindStringSubmatch(string(peekData))
+		p.debug("readPythonMultilines: matched ", len(peekMatches), " parts:")
+		for n, v := range peekMatches {
+			p.debug("   ", n, ": '", v, "'")
+		}
+
+		// Return if not a Python multiline value.
 		if len(peekMatches) != 3 {
+			p.debug("readPythonMultilines: didn't match enough parts, meaning the value has ended")
+			p.debug("readPythonMultilines: value is: '"+line+"'")
```

Comment:
One single debug line should be self-contained, it can be very long and I don't think that will be a problem.

Reply:
I don't think this is necessary, but whatever, I'll make the change to close this out